cltk/utils/philology.py

```diff
@@ -45,8 +45,9 @@ def _build_concordance(self, text_str):
         #! rm dupes after index, before loop
         tokens = set(orig_tokens)
         tokens = [x for x in tokens if x not in [',', '.', ';', ':', '"', "'", '[', ']']]  # this needs to be changed or rm'ed
-
-        return c.return_concordance_all(tokens)
+        readable_tokens = ["#_" + aToken.swapcase() + "_#" for aToken in tokens] # Changed Kaan
```

Comment:
Small note: You don't need to add a `# Changed Kaan`, as Git shows me the diffs automatically.